wolfcrypt/test/test.c

```diff
@@ -4005,6 +4017,156 @@ int aes_test(void)
     return ret;
 }
 
+int aes192_test(void)
+{
+#ifdef HAVE_AES_CBC
+    Aes enc;
+    byte cipher[AES_BLOCK_SIZE];
+#ifdef HAVE_AES_DECRYPT
+    Aes dec;
+    byte plain [AES_BLOCK_SIZE];
+#endif
+#endif /* HAVE_AES_CBC */
+    int  ret = 0;
+
+#ifdef HAVE_AES_CBC
+    /*
+     * http://www.inconteam.com/software-development/41-encryption/
+     *                                         55-aes-test-vectors#aes-cbc-192
+     */
```

Comment:
This link keeps redirecting me to adds, not actual aes-test-vectors. The vectors are NIST ones, so I recommend just removing this link reference. Same with the one below.